libcloud/compute/drivers/openstack.py

```diff
@@ -1449,9 +1449,19 @@ class OpenStack_1_1_NodeDriver(OpenStackNodeDriver):
     _networks_url_prefix = "/os-networks"
 
     def __init__(self, *args, **kwargs):
+        self.ex_force_microversion = str(kwargs.pop("ex_force_microversion", ''))
```

Comment:
For consistency with ``ex_force_api_version``, it may be better to default it to ``None``.